gui-apps/aylurs-gtk-shell/aylurs-gtk-shell-1.8.2.ebuild

```diff
@@ -54,6 +54,12 @@ src_configure() {
 	meson_src_configure
 }
 
+src_install() {
+	meson_install
+	rm ${D}/usr/bin/ags
+	ln -s /usr/share/com.github.Aylur.ags/com.github.Aylur.ags ${D}/usr/bin/ags
```

Comment:
Please fix [`post_install.sh`](https://github.com/Aylur/ags/blob/bb91f7c8fdd2f51c79d3af3f2881cacbdff19f60/post_install.sh#L12) instead. Either make `/usr/bin/ags` a relative symlink or fix the absolute symlink such that `$BIN_SRC` doesn't contain `$DESTDIR`.
This is an upstream bug that should be reported.